requirements/base.txt

```diff
@@ -156,7 +156,7 @@ pydispatcher==2.0.5
     # via scrapy
 pyopenssl==21.0.0
     # via scrapy
-pyparsing==3.0.3
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` requires `pyparsing<3` (see change https://github.com/pypa/packaging/pull/471). This constraint required will be fixed in the next version.